deno/payloads/v8/_interactions/_applicationCommands/chatInput.ts

```diff
@@ -217,6 +219,11 @@ export interface ApplicationCommandInteractionDataOptionNumber
 	focused?: boolean;
 }
 
+export type ApplicationCommandInteractionDataOptionAttachment = InteractionDataOptionBase<
+	ApplicationCommandOptionType.Attachment,
+	APIAttachment
```

Comment:
Isn't this an id?

Reply:
oh shit my bad, yes this is a snowflake id I had kept it as an api attachment when I was still unsure of the structure. 
Fixed it now